xbmc/utils/LangCodeExpander.cpp

```diff
@@ -234,6 +242,78 @@ bool CLangCodeExpander::ConvertToISO6392T(const std::string& strCharCode,
   return false;
 }
 
+// Lookup ISO-3166-1 table and return codes or name.
+bool CLangCodeExpander::LookupISO31661(std::string_view saught,
```

Comment:
saught? meant to be sought? Might want to use something simpler (searchString, searchItem, etc)

Reply:
Done